### PR TITLE
Error Messages not clear for Adopting suspended stacks - Issue #69

### PR DESCRIPTION
**Pull request by owen-lynch-ibm**
To address https://github.com/IBM/openstack-vim-driver/issues/69

Original internal dev issue:
https://github.ibm.com/TNC/tnc-o-tracking/issues/222

**Change description**

The error message from Openstack driver used to be just the `stack_status_reason` field.
However, this expands that to a more user friendly message containing stack status and reason, for *Adopt* transitons that fail.

### CODE REVIEW

File: tests/unit/service/test_resourcedriver.py [hunk 1]
```diff
@@ -454,7 +454,7 @@ def test_get_lifecycle_execution_adopt_suspended(self):
         self.assertIsInstance(execution, LifecycleExecution)
         self.assertEqual(execution.request_id, 'Adopt::1::request123')
         self.assertEqual(execution.status, 'FAILED')        
-        self.assertEqual(str(execution.failure_details), 'failure_code: INFRASTRUCTURE_ERROR description: SUSPEND_COMPLETE')
+        self.assertEqual(str(execution.failure_details), 'failure_code: INFRASTRUCTURE_ERROR description: Adopt Failed. Stack Status: SUSPEND_COMPLETE Reason: SUSPEND_COMPLETE')
```
Comment (sglover) [inline tests/unit/service/test_resourcedriver.py:457]: This error message is still not very clear to me. Perhaps something like `Adopt failed: cannot adopt stack with status SUSPEND_COMPLETE` would read better?

Comment (owen-lynch-ibm) [reply tests/unit/service/test_resourcedriver.py:457]: OK so not including the `stack_status_reason`  (which I agree can look a bit cumbersome)
and instead it is:

`Adopt failed: cannot adopt stack with status + <STACK_STATUS>`

yeah I agree with this - I did think the reason was a bit too wordy

Comment (owen-lynch-ibm) [reply tests/unit/service/test_resourcedriver.py]: Done

**Review by sglover — APPROVED** [review]

**Review by eflamini — APPROVED** [review]
LGTM
